package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -25,17 +25,18 @@
 Nucleic acid analysis --- :mod:`MDAnalysis.analysis.nuclinfo`
 =============================================================
 
-:Author: Elizabeth Denning
-:Year: 2011
+:Authors: Elizabeth Denning, Abhishek A. Kognole
```

Comment:
Only just picked this up - @orbeckst I've never been super clear on how these authorship entries worked in MDA, is this the correct way to address this? (and also do we have a clear view on when we add someone and when we don't?)

Reply:
No real guide lines and I would be liberal with it for right now.

Truth be told, I'd actually like to get rid of these entries (which come from the earliest days...) to make the docs more uniform and because AUTHORS is really the most appropriate part for noting contributions as it makes clear that this is a collective work.

So, I'd just leave it as is, with the understanding that all of these entries will maybe suddenly disappear .